src/v2/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkSharePanel.tsx

```diff
@@ -128,9 +128,11 @@ export class ArtworkSharePanel extends React.Component<
               onClick={this.handleCopy}
             />
           </SansGrow>
-          <Sans size={["3", "2"]} weight="medium" color="black60">
+          <Text variant="mediumText" color="black60">
+            {/* FIXME Remove lint ignore */}
+            {/* eslint-disable-next-line  */}
             <a onClick={this.handleCopy}>{this.state.copyLabelText}</a>
```

Comment:
You *should* be able to just swap these with `Clickable` and it work out just fine.

Reply:
Does the onClick move to the clickable?

Reply:
Yeah, `Clickable` is just a button with default styling turned off and box mixed in.